src/server/game/Movement/PathGenerator.cpp

```diff
@@ -482,8 +482,9 @@ void PathGenerator::BuildPointPath(const float *startPoint, const float *endPoin
         memcpy(&pathPoints[VERTEX_SIZE * 0], startPoint, sizeof(float)* 3); // first point
 
         // path has to be split into polygons with dist SMOOTH_PATH_STEP_SIZE between them
-        G3D::Vector3 startVec = G3D::Vector3(startPoint[0], startPoint[1], startPoint[2]);
-        G3D::Vector3 endVec = G3D::Vector3(endPoint[0], endPoint[1], endPoint[2]);
+		//startPoint and endPoint are in format y,z,x while G3D::Vector3 expects x,y,z
```

Comment:
I know it uses yzx but that's not the case, the 2 vectors being created _DO NOT_ get used in any recast function, which is why it's wrong, it straight up copies them to _pathPoints which DOES expects x y z while the format is still yzx, which is invalid : 
`while (len > SMOOTH_PATH_STEP_SIZE)
        {
            len -= SMOOTH_PATH_STEP_SIZE;
            prevVec += diffVec;
            pathPoints[VERTEX_SIZE * pointCount + 0] = prevVec.x;
            pathPoints[VERTEX_SIZE * pointCount + 1] = prevVec.y;
            pathPoints[VERTEX_SIZE * pointCount + 2] = prevVec.z;
            ++pointCount;
        }`